baseframe/forms/validators.py

```diff
@@ -319,6 +320,19 @@ def __call__(self, form, field):
             return
         else:
             raise ValidationError(self.message)
+            
+class IsEmoji(object):
+    """
+    Validate whether the supplied string is an emoji.
+    """
+    def __init__(self, message=None):
+        self.message = message
+
+    def __call__(self, form, field):
+        if field.data in emoji.UNICODE_EMOJI:
+            return
```

Comment:
No need for this `return`. Check for the invalid case and raise `ValidationError` if needed.